src/features/auth/signup/QuitSignupModal.tsx

```diff
@@ -14,16 +15,30 @@ import { Spacer } from 'ui/components/spacer/Spacer'
 import { Warning } from 'ui/svg/icons/Warning'
 import { ColorsEnum, Typo } from 'ui/theme'
 
+export enum SignupSteps {
```

Comment:
Le typage sans enum me serait aussi aller, mais why not

Reply:
Typescript préfère cette solution, quand je mettais `'Email' | 'Password' | 'Birthday' | 'CGU'` directement et que je donnais une  string en prop, il y avait une erreur ` Type 'string' is not assignable to type '"Birthday" | "Email" | "Password" | "CGU"'`